src/store/rescan.js

```diff
@@ -29,6 +29,7 @@ export default {
       return start(rootState.auth)
         .then(result => {
           result.running = true;
+          result.last_click = new Date();
```

Comment:
This value is only set after scan is triggered, so I'm not sure why we would do this. If the user hasn't triggered a scan, this is `undefined` and will always be false in comparison to `rescan.finished_at`

Reply:
Ah, read that wrong. No problem then. I do think setting a default value on app startup is a good idea though, because I don't like relying on comparisons between different types.